CAN-1317, CAN-1321

vikasranatalentelgia opened the pull request:
Filed name on User profile must be uniform
Account Settings: UI issues
Support order issue

vikasranatalentelgia committed:
CAN-1317, CAN-1321

Filed name on User profile must be uniform
Account Settings: UI issues
Support order issue

akuks commented on src/components/ComponentPages/UserProfile/UserProfileDetails/UserProfileCard.tsx:
```diff
@@ -92,7 +92,7 @@ export const UserProfileCard = ({
                                           {" "}
-                                          (Support delegated to{" "}
+                                          (Support Delegated to{" "}
                                           <a className={styles.Bluecolor}>
```
Aren't we using `<Link>` here?

vikasranatalentelgia replied:
Done

vikasranatalentelgia committed:
Merge branch 'development' into Fixes

vikasranatalentelgia committed:
CAN-1331

User is able to delegate their support to their delegated supporter

vikasranatalentelgia committed:
Merge branch 'development' into Fixes

vikasranatalentelgia committed:
CAN-1291,CAN-1338

Account Settings: Disabled fields in Profile Info can be edited from DOM
User is not added as direct supporter in below case

vikasranatalentelgia committed:
CAN-1343,CAN-1345

Upload File: Search filter doesn't reset
"Your supporting camps list for topic <topic-name> is missing on support page

vikasranatalentelgia committed:
Build issue fixed

Build issue fixed

vikasranatalentelgia committed:
Build issue resolved

Build issue resolved
